### ISSUE TITLE
Synchronization between camera images and camera pose 

### ISSUE THREAD

**Issue by Amigoshan** [microsoft/AirSim#1309]
I encountered  the synchronization issue similar to [this](https://github.com/Microsoft/AirSim/issues/275) post. 

What I want to get from the simulation is synchronized RGB and depth image, along with the camera pose. But I found they are not well synchronized. 

I'm using python API simGetImages. According to [issue 275](https://github.com/Microsoft/AirSim/issues/275), this should return two images that perfectly synchronized. I also get the camera pose from response.camera_position/orientation. I print out the timestamp, this is what it looks like:

![image](https://user-images.githubusercontent.com/11531951/43682916-e1e0fdd6-984f-11e8-8195-fcfee76235ba.png)

From the above, the time stamp of those three images are slightly different. However the camera poses returned by simGetImages for those three images are exactly the same. 

I'm trying to do image warping. Between each two consecutive frames (RGB1, Depth1, Pose1), (RGB2, Depth2, Pose2). I use RGB1, Depth1, Pose1 and Pose2, calculate RGB2_warp. If all of those inputs are well synchronized, my output RGB2_warp should be well aligned with RGB2. But there's a big error when the drone is changing it's speed. My observation is that the camera pose is a little bit behind the images. 

![image](https://user-images.githubusercontent.com/11531951/43683026-b93916d2-9851-11e8-8e7b-cfb2482b6036.png)

I also tried the [record mode](https://github.com/Microsoft/AirSim/blob/master/docs/settings.md#recording), from the airsim_rec.txt, the timestamp differs even more: (according to the 'Timestamp' column and timestamp in ImageName)

![image](https://user-images.githubusercontent.com/11531951/43683049-4ec8a80c-9852-11e8-9c5c-d0bbb7dcc083.png)

In summary, I wonder how to get images and camera pose that are perfectly synchronized?  

Thanks!
